android/README.md

```diff
@@ -2,9 +2,18 @@
 
 ## Prerequisites
 * Android Studio 3.3.2+
-* Java SE Development Kit 8
+* Java SE Development Kit 11 (Java 9 or newer)
 * [Node.js](https://nodejs.org/) 8.9+ (for building KeymanWeb)
 
+## Install Java
+It is recommended to use openJDK because of oracle license issues.
+Tested with latest release for openJDK 11 from
+https://github.com/ojdkbuild/ojdkbuild
+
+1. Download and unpack the zip archive
+2. On on windows: use the default java path C:\Program Files\Java to avoid error message "Error 0x80010135 Path Too Long".
```

Comment:
```suggestion
2. On Window, install Java to the default path "C:\Program Files\Java" to avoid the error message `Error 0x80010135 Path Too Long`.
```